### ISSUE TITLE
Web UI error which leads to degraded functionality

### ISSUE THREAD

**Issue by Timer** [rethinkdb/rethinkdb#4261]
```
Uncaught TypeError: Cannot read property 'length' of undefined
    Container.Container.set_doc_description @ cluster-min.js?v=2.0.2:5156
    Container.Container.set_docs @ cluster-min.js?v=2.0.2:5208
    (anonymous function) @ cluster-min.js?v=2.0.2:12039
    f.Callbacks.o @ jquery-1.7.2.min.js?v=2.0.2:2
    f.Callbacks.p.fireWith @ jquery-1.7.2.min.js?v=2.0.2:2
    e.extend.ready @ jquery-1.7.2.min.js?v=2.0.2:2
    c.addEventListener.B @ jquery-1.7.2.min.js?v=2.0.2:2
```

**Comment by AtnNn** [comment]
This seems to be caused by the updated docs in 2.0.2

**Comment by danielmewes** [comment]
Oh no I broke the auto completion. Sorry. :-(
@AtnNn when you find a fix for this, would you mind doing another release? Sorry for the extra work.

**Comment by AtnNn** [comment]
Fixed in a CR for @deontologician and in `atnnn/4261` (55727c9)

**Comment by AtnNn** [comment]
Fixed in 7f7ee03
